### DISCUSSION TITLE
How to contribute to opensource

### DISCUSSION THREAD

**Discussion by Maniik113** [community/community#72599] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I am new to GitHub and I want to know how to contribute to open source.

**Comment by Maniik113** [comment]
@Manishak798

**Reply by Manishak798** [reply]
check

**Comment by Manishak798** [comment, marked as answer]
1. **Create a GitHub Account**: If you don't have one, sign up on GitHub.

2. **Find a Project**: Look for a project you want to help with. It could be something you use or are interested in.

3. **Fork the Repository**: Click the "Fork" button on the top-right of the project's GitHub page. This makes a copy of the project in your GitHub account.

4. **Clone Your Fork**: Use Git to make a copy of your fork on your computer.

5. **Create a Branch**: Create a new branch in your fork to work on your changes.

6. **Make Your Changes**: Modify the code or files in your branch.

7. **Commit Your Changes**: Save your changes with a brief description.

8. **Push Your Changes**: Send your changes back to your GitHub fork.

9. **Create a Pull Request (PR)**: On the original project's page, click "New Pull Request" to propose your changes. Explain what you did.

10. **Wait for Review**: The project maintainers will review your PR. They might ask for changes.

11. **Respond to Feedback**: If needed, make more changes and push them to your branch.

12. **PR Gets Merged**: If your changes are accepted, the maintainers will merge them into the project.

13. **Congratulations!**: You've contributed to open source!

That's the basic process in simple terms. Remember, open source is about collaboration and sharing, so don't be afraid to ask for help or offer assistance to others in the community.

**Comment by Maniik113** [comment]
Thankyou
